applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -110,26 +119,50 @@ static int stream_index_get(struct bt_audio_stream *stream, uint8_t *index)
 	return -EINVAL;
 }
 
-static int stream_index_from_conn_get(struct bt_conn *conn, uint8_t *index)
+static bool all_active_headsets_started(void)
```

Comment:
```suggestion
static bool all_headsets_streaming(void)
```

Reply:
Disagree, the 'active' part of the function name is an important distinction and explanation of what this function does. The headset is only active if it has a sink stream. Could potentially add 'sinking' in there, but I felt that was a bit too verbose.

Reply:
Added doc for the function

Reply:
I think it is hard to understand for a person reading this code for the first time that a headset is only active if it has a sink stream. Even if a device/headset is only sending audio to the gateway it is still "active" in my mind.
Maybe change to something like:
```suggestion
static bool all_headset_sinks_started(void)
```